Sources/TensorFlow/Core/LazyTensorShapeInference.swift

```diff
@@ -47,15 +47,30 @@ extension LazyTensorOperation {
         let status = TF_NewStatus()
         defer { TF_DeleteStatus(status) }
 
-        let inputShapes: [TensorShape] = inputs.lazy.flatMap { (input: Input) -> [TensorShape] in
+        /// Returns shape only if it has already been computed.
+        func shapeIfAvailable(_ handle: LazyTensorHandle) -> TensorShape? {
```

Comment:
```suggestion
        func shape(for handle: LazyTensorHandle) -> TensorShape? {
```
1. The `IfAvailable` semantics is already implied by the optional return type.
2. Then handle needs an argument label to form a phrase with the method base name.

Reply:
Thanks! Done.